packages/web/style.css

```
@@ -5,53 +5,114 @@
   --ak-rail-gap: 24px;
   --ak-bubble-gap: 8px;
-  --ak-line-height: 1.65;
-  --ak-block-gap: 2px;
-  /* The hand-off between the gutter and the text. The gutter pays it back in negative
-     margin to reach the text, so the two must move together or the dead gap comes back */
+  --ak-line-height: 1.8;
+  --ak-block-gap: 3px;
   --ak-gutter-gap: 8px;
-  --ak-para-gap: 6px;
+  --ak-para-gap: 7px;
   --ak-font: -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", "Hiragino Sans", "Noto Sans JP", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)